src/components/Download/DownloadOptionsForm/AdvanedOptions.js

```diff
@@ -17,7 +18,9 @@ export const AdvanedOptions = ({
   const skipQuantileNormalize = !quantileNormalize
 
   useEffect(() => {
-    setToggle(skipQuantileNormalize)
+    if (setToggle) {
+      setToggle(skipQuantileNormalize)
+    }
```

Comment:
Instead of this, let's just default `setToggle` to a noop

Reply:
To make the file simpler, instead, I've deleted the prop `setToggle` and adjusted the codebase at the commit 1e9690c1a28fceb77aa864ebcccc23a27924ed9c.